tests/attested_tls/common/settings.c

```diff
@@ -0,0 +1,55 @@
+// Copyright Microsoft.
+// Licensed under the attached Microsoft Software License Terms
+
+#include "settings.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "log_utils.h"
+
+char* get_environment_variable(const char* name)
+{
+    const size_t buf_size = 128;
+    char str[buf_size];
+    int res = 0;
+
+    if (name == NULL)
+    {
+        goto fail;
+    }
+
+    if (getenv(name) == NULL)
```

Comment:
Can you remove the whole if block here? This would pause the pipeline if the env variables do not exist.

Reply:
I think there is a timeout value, but it make sense to me to remove the entire if block.